### PR TITLE
Major Readme Overhaul

### PR DESCRIPTION
**Pull request by Okazakee**
Deleted useless redundant stuff and obsolete notations, better layout even for future changes, there is other room for improvements.

### CODE REVIEW

**Review by imsofi — CHANGES_REQUESTED** [review]
Lets not delete history here. This has become a core resource to find out what things are and were problems, how those issues were fixed, and any of their statuses plus further fixes.

Also most of the things you have deleted are still relevant issues, all without upstream patches? Like the USB-C dock is not up-streamed, and likely will not be fixed until 6.2. S4 sleep is still relevant, "the speakers suck" is still relevant, screen rotation is not yet up-streamed and has no ETA, S3 sleep is still relevant. I can go on but i think the general idea comes across.

File: README.md [hunk 1]
```diff
@@ -1,66 +1,60 @@
...
+## Latest Fixes:
+> **Kernel independent:**
+ > - [x] **S4 sleep | Hibernating (guide is still missing)**
+ > - As an alternative for S3 sleep, S4 sleep can be made use of (requires a SWAP partition).
+ > - [x] **Screen rotation**
+ > - Automatic rotation does not work out of the box in GNOME for some reason.
+ > - Can be fixed with an [extension](https://github.com/shyzus/gnome-shell-extension-screen-autorotate).
+ 
+> **Kernel 6.1+:**
+ > - [x] **Fixed incompatibility with 'Delock' USB-C dock**
```
Comment (imsofi) [inline README.md:22]: This is not yet fixed, see #13

Comment (imsofi) [reply README.md:22]: Will likely only get properly fixed in 6.2, some docks can still work under 6.1.

**Comment by tomsom** [comment]
Neat! Thank you :))

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ _**For more information take a look at the [issues](https://github.com/tomsom/yo
 
 ## Latest Fixes:
 > **Kernel independent:**
+ > - [x] **Fingerprint Sensor**
+ > - Seems to be working with the driver from this [Copr repo](https://copr.fedorainfracloud.org/coprs/d-k-bo/libfprint-goodixtls/).
```
Comment (imsofi) [inline README.md:14]: This is still heavy WIP, not stable, and not cross platform. Should probably be put into a "Workaround" section to show the significance of not being a proper fix yet.

**Review by imsofi — COMMENTED** [review]
Added some comments on places that need some rewording, overall good once those are changed up. :+1:

**Comment by imsofi** [comment]
Ah shoot this got merged while i reviewed, guess we should get these issues into a new PR. :sweat_smile:

**Comment by tomsom** [comment]
I don't have a model with a fingerprint reader so id be cool if you peeps can test what works and what not..

**Comment by Myaats** [comment]
Might be worth adding a disclaimer for the BG5 firmware issue on models that use the BG5 SSD instead of Samsung, Micron or WD that permanently bricks unless they are updated to the newest firmware (0107ANLS) through Windows Update (It is not available through fwupd).

**Comment by tomsom** [comment]
@Myaats Oh yes, should have added this a long time ago :o
Thank you for the reminder!

**Comment by Okazakee** [comment]
Been a pleasure

**Pull request merged**